src/main/java/com/adobe/commerce/cif/api/HealthApi.java

```diff
@@ -0,0 +1,57 @@
+/*******************************************************************************
+ *
+ *    Copyright 2018 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+
+package com.adobe.commerce.cif.api;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+import static com.adobe.commerce.cif.api.Constants.HTTP_OK;
+import static com.adobe.commerce.cif.api.Constants.HTTP_OK_MESSAGE;
+import static com.adobe.commerce.cif.api.Constants.HTTP_SERVICE_UNAVAILABLE;
+import static com.adobe.commerce.cif.api.Constants.HTTP_SERVICE_UNAVAILABLE_MESSAGE;
+
+@Path("/_health")
+@Api(value = "/_health")
+@Produces(MediaType.TEXT_PLAIN)
```

Comment:
Why not JSON? I all services, even this which is not for storefront/end users should stick with JSON. That is documented here: https://github.com/adobe/commerce-cif-api#api

Reply:
The action is not supposed to return anything in the body and the default content type in HTTP is `text/plain`.

Reply:
@masodu This needs to be changed to json after your recent changes.